lib/sendf.c

```diff
@@ -405,8 +405,11 @@ CURLcode Curl_write_plain(struct Curl_easy *data,
                           ssize_t *written)
 {
   CURLcode result;
-  struct connectdata *conn = data->conn;
-  int num = (sockfd == conn->sock[SECONDARYSOCKET]);
+  struct connectdata *conn;
+  int num;
+  conn = data->conn;
```

Comment:
you could leave it as `struct connectdata *conn = data->conn;`

Reply:
Oops, yes I can!